src/components/pages/donation_form/SubPages/DonationForm.vue

```diff
@@ -25,19 +25,29 @@
 			@set-address-type="setAddressType( $event )"
 		/>
 
-		<div class="content-card">
-			<FormSummary>
-				<template #summary-content v-if="paymentSummary">
-					<DonationSummary
-						:address="addressSummary"
-						:payment="paymentSummary"
-						:bank-data="bankDataSummary"
-						:countries="countries"
-						:salutations="salutations"
-					/>
-				</template>
-
-				<template #summary-buttons>
+		<ContentCard>
+			<template #heading v-if="paymentSummary">
+				<h2>{{ $t( 'donation_form_summary_title' ) }}</h2>
```

Comment:
@Abban In the browser, the faded line (which is there because of the `<hr/>` in `SectionHeading` component) is always showing even when there is no "Summary headline".

Reply:
I answered this in a previous change request.